### ISSUE TITLE
Multiplicity (and control parameter) handling

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#2243]
**Describe the bug**
sasmodels implements `control` parameters which are `hidden`. They are documented in the developer docs, in particular at:
https://www.sasview.org/docs/dev/sasmodels-api/sasmodels.html#submodules

However, the GUI does not interpret/use them correctly. These are used in the so-called `multiplicity` models which includes the RPA, the core_multishell, the unified_power_Rg, and the onion models.  The RPA is different from the others in that it is really a series of cases rather than a number of shells. While there are issues with all, the RPA is so broken in the GUI that it has been removed form version 5.

**To Reproduce**

**Expected behavior**

**Screenshots**

**SasView version (please complete the following information):**
 - Version: [e.g. 5.0.4]

**Operating system (please complete the following information):**
 - OS: [e.g. Windows 10]

**Additional context**

**Comment by RichardHeenan** [comment]
Additionally, adding S(Q) to say core_multi_shell does not give the correct results, nor does it give any error messages to the user. The attached project file will show this.  The fits on the two plots below should, if I got the models right, be the same! 

The core_multi_shell is not adding separate P(Q) and s(Q) to the data explorer.

More worrying still, if you remove the S(Q) and then fit just "scale" the core_multi_shell still does not give quite the same result as core_shell_sphere (no plots here). How different they are depends on the value of the flat background. I suspect that for core_multi_shell the flat background is perhaps not being included in the chi squared (though is there in the I(Q) on the plot).

Basically the gui seems to have a lot of issues with multiplicity models, which will not be apparent to the average user.

![Screenshot 2023-09-19 114154](https://github.com/SasView/sasview/assets/4226214/670ac888-3d6c-4cff-acfe-8bd0d8036e55)

![Screenshot 2023-09-19 114532](https://github.com/SasView/sasview/assets/4226214/2f38efe1-5fab-4ea5-83ab-a34ffda15efe)
![Screenshot 2023-09-19 114614](https://github.com/SasView/sasview/assets/4226214/e72da27f-fd93-4712-917b-bcafff528f4d)


[test_multishellSQ_19Sept2023_506release.json.txt](https://github.com/SasView/sasview/files/12658979/test_multishellSQ_19Sept2023_506release.json.txt)

**Comment by smk78** [comment]
As far as the _rpa_ model is concerned, the consensus at the 2024 Contributor Camp roadmap discussion was that this model **should not** be re-implemented in the form it once was (ie, using a control variable to select the calculation case). Instead, each case should be implemented as a separate model.

**Comment by butlerpd** [comment]
In closing issue #1089 with the merging of PR #2647, @pkienzle's comment about replacing the old multiplicity concept being maintained in sasview with sasmodels' more sophisticated `control` and `hidden` parameters is copied over here.

> Could redo the sasview model table view so that parameter list is updated whenever any control parameter is updated (sasmodels parameters each have an "is_control" attribute), and drop the notion of a "multiplicity" model.
